std/range/package.d

```diff
@@ -8904,11 +8904,12 @@ struct NullSink
 /++
 
   Implements a "tee" style pipe, wrapping an input range so that elements of the
-  range can be passed to a provided function or $(LREF OutputRange) as they are
+  range can be passed to a provided function or
+  $(XREF std, range, interfaces, OutputRange) as they are
   iterated over. This is useful for printing out intermediate values in a long
   chain of range code, performing some operation with side-effects on each call
   to $(D front) or $(D popFront), or diverting the elements of a range into an
-  auxiliary $(LREF OutputRange).
+  auxiliary $(XREF std, range, interfaces, OutputRange).
```

Comment:
No reason that it can't be done.  I just meant that I was only trying to fix broken links so I didn't look at changing it.